### PR TITLE
Merge 20180130

### PR DESCRIPTION
**Pull request by arjantijms**
Merged master (Payara 4) into the Payara-5 branch; 2018-01-30 ~15:15

### CODE REVIEW

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

File: appserver/transaction/jta/src/main/java/com/sun/enterprise/transaction/JavaEETransactionManagerSimplified.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
  * holder.
  */
 
-// Portions Copyright [2016-2017] [Payara Foundation and/or its affiliates.]
+// Portions Copyright [2016] [Payara Foundation and/or its affiliates.]
```
Comment (Pandrex247) [inline appserver/transaction/jta/src/main/java/com/sun/enterprise/transaction/JavaEETransactionManagerSimplified.java]: Well this change doesn't seem right...

**Comment by lprimak** [comment]
jenkins quick test payara 5

**Pull request merged**